internal/cmd/hana/check.go

```diff
@@ -17,31 +16,33 @@ import (
 )
 
 type hanaCheckConfig struct {
-	ctx        context.Context
-	kubeClient kube.Client
+	*cmdcommon.KymaConfig
+	cmdcommon.KubeClientConfig
 
-	kubeconfig string
-	name       string
-	namespace  string
-	timeout    time.Duration
+	name      string
+	namespace string
+	timeout   time.Duration
 }
 
-func NewHanaCheckCMD() *cobra.Command {
-	config := hanaCheckConfig{}
+func NewHanaCheckCMD(kymaConfig *cmdcommon.KymaConfig) *cobra.Command {
+	config := hanaCheckConfig{
+		KymaConfig:       kymaConfig,
+		KubeClientConfig: cmdcommon.KubeClientConfig{},
+	}
 
 	cmd := &cobra.Command{
 		Use:   "check",
 		Short: "Check if the Hana instance is provisioned.",
 		Long:  "Use this command to check if the Hana instance is provisioned on the SAP Kyma platform.",
 		PreRunE: func(_ *cobra.Command, args []string) error {
-			return config.complete()
+			return config.KubeClientConfig.Complete()
 		},
 		RunE: func(_ *cobra.Command, _ []string) error {
 			return runCheck(&config)
 		},
 	}
 
-	cmd.Flags().StringVar(&config.kubeconfig, "kubeconfig", "", "Path to the Kyma kubecongig file.")
+	config.KubeClientConfig.AddFlag(cmd)
```

Comment:
IMO now we could remove `_ = cmd.MarkFlagRequired("kubeconfig")` or move it into `kcc.AddFlag()`

Reply:
yes, I forget about this...